### PR TITLE
Adding versioning for D track

### PR DESCRIPTION
**Pull request by amscotti**

### CODE REVIEW

File: src/data/tracks.json [hunk 1]
```diff
@@ -81,7 +81,10 @@
   {
     "slug": "d",
     "name": "D",
-    "core_enabled": false
+    "core_enabled": false,
+    "versioning": "exercises/{slug}/.meta/version",
+    "stub_file": "exercises/{slug}/source/{exercise_slug}.d",
+    "test_file": "exercises/{slug}/source/{exercise_slug}.d"
```
Comment (SleeplessByte) [inline src/data/tracks.json]: Are you _certain_ the test file is the same as the stub file? Do tests live in the source file in D?

Comment (amscotti) [reply src/data/tracks.json]: Yes, they do. D is a bit odd like that but the code and the unit test need to be in the same file.

Comment (SleeplessByte) [reply src/data/tracks.json]: Alright thanks! cc @ihid note that D doesn't allow for exercise updates

**Comment by SleeplessByte** [comment]
Will deploy this shortly!

File: src/data/tracks.json (hunk 1 above)
Comment (iHiD) [reply src/data/tracks.json]: Thanks both.

**Pull request merged**